IDEAS/LIDEAS/Examples/ZoneLinearise.mo

```diff
@@ -0,0 +1,187 @@
+    annotation (Placement(transformation(extent={{-30,-100},{-20,-80}})));
+  IDEAS.Buildings.Components.OuterWall Roof(
+    azi=0,
+    A=10,
+    redeclare Buildings.Validation.Data.Constructions.LightRoof
+      constructionType,
+    inc=IDEAS.Types.Tilt.Ceiling) "Outer wall model"
+    annotation (Placement(transformation(extent={{-2,-100},{8,-80}})));
+equation
+  connect(commonWall.propsBus_a, groFlo.propsBus[1]) annotation (Line(
+      points={{-44.8333,10},{-12,10},{-12,-4.4},{20,-4.4}},
+      color={255,204,51},
+      thickness=0.5,
+      smooth=Smooth.None));
+  connect(window.propsBus_a, firFlo.propsBus[2]) annotation (Line(
+      points={{-44.8333,-70},{6,-70},{6,-55.5},{20,-55.5}},
+      color={255,204,51},
+      thickness=0.5,
+      smooth=Smooth.None));
+  connect(outerWall.propsBus_a, groFlo.propsBus[4]) annotation (Line(
+      points={{-44.8333,-46},{-12,-46},{-12,-6.8},{20,-6.8}},
+      color={255,204,51},
+      thickness=0.5,
+      smooth=Smooth.None));
+  connect(slabOnGround.propsBus_a, groFlo.propsBus[3]) annotation (Line(
+      points={{-44.8333,32},{-12,32},{-12,-6},{20,-6}},
+      color={255,204,51},
+      thickness=0.5,
+      smooth=Smooth.None));
+  connect(shadingControl.y, window.Ctrl) annotation (Line(
+      points={{-60,-84},{-58,-84},{-58,-86},{-52.3333,-86},{-52.3333,-82}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(commonWall1.propsBus_a, groFlo.propsBus[5]) annotation (Line(
+      points={{-44.8333,-14},{-12,-14},{-12,-7.6},{20,-7.6}},
+      color={255,204,51},
+      thickness=0.5));
+  connect(const.y, commonWall.T)
+    annotation (Line(points={{-79,10},{-57.8333,10}},
+                                                   color={0,0,127}));
+  connect(const1.y, commonWall1.Q_flow) annotation (Line(points={{-79,-20},{
+          -57.8333,-20},{-57.8333,-18}},
+                                   color={0,0,127}));
+  connect(outerWall1.propsBus_a, firFlo.propsBus[3]) annotation (Line(
+      points={{-20.8333,-88},{-14,-88},{-14,-72},{8,-72},{8,-56.5},{20,-56.5}},
+      color={255,204,51},
+      thickness=0.5));
+
+  connect(Roof.propsBus_a, firFlo.propsBus[4]) annotation (Line(
+      points={{7.16667,-88},{20,-88},{20,-57.5}},
+      color={255,204,51},
+      thickness=0.5));
+  connect(floor.propsBus_b, groFlo.propsBus[2]) annotation (Line(
+      points={{7,-33.8333},{7,-5.2},{20,-5.2}},
+      color={255,204,51},
+      thickness=0.5));
+  connect(floor.propsBus_a, firFlo.propsBus[1]) annotation (Line(
+      points={{7,-42.1667},{7,-54.5},{20,-54.5}},
+      color={255,204,51},
+      thickness=0.5));
+  annotation (Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,
+            -100},{100,100}})),           __Dymola_Commands(file=
+          "Scripts/linearize_ZoneLinearise.mos" "Linearise"),
+    Documentation(revisions="<html>
+<ul>
+<li>August 21, 2018 by Damien Picard: <br/>Change medium to <code>IDEAS.Media.Specialized.DryAir</code>.</li>
```

Comment:
Can you spread the revision history across multiple lines in the future like the IBPSA convention?
```
<li>
August 21, 2018 by Damien Picard: <br/>
Change medium to <code>IDEAS.Media.Specialized.DryAir</code>.
</li>
```